src/gen9_avc_encoder.c

```diff
@@ -3596,6 +3662,456 @@ gen9_avc_set_curbe_mbenc(VADriverContextP ctx,
 }
 
 static void
+gen9_avc_fei_set_curbe_mbenc(VADriverContextP ctx,
+                             struct encode_state *encode_state,
+                             struct i965_gpe_context *gpe_context,
+                             struct intel_encoder_context *encoder_context,
+                             void * param)
+{
+    struct i965_driver_data *i965 = i965_driver_data(ctx);
+    gen9_avc_fei_mbenc_curbe_data *cmd;
+    struct encoder_vme_mfc_context * vme_context = (struct encoder_vme_mfc_context *)encoder_context->vme_context;
+    struct generic_enc_codec_state * generic_state = (struct generic_enc_codec_state *)vme_context->generic_enc_state;
+    struct avc_enc_state * avc_state = (struct avc_enc_state *)vme_context->private_enc_state;
+    VASurfaceID surface_id;
+    struct object_surface *obj_surface;
+    VAEncSliceParameterBufferH264 * slice_param = avc_state->slice_param[0];
```

Comment:
Yeah,  would better we can have a rule for the unified coding style.  Before Haihao do this, you can ignore my comment. : )